examples/CMakeLists.txt

```diff
@@ -28,8 +28,8 @@ if (THRUST_ENABLE_EXAMPLE_FILECHECK)
   endif()
 
   execute_process(
-    COMMAND "${THRUST_FILECHECK_EXECUTABLE}" "${filecheck_data_path}/thrust.sanity.filecheck"
```

Comment:
Did this configure successfully when you tested it? I believe that this file in `internal/test/` will need to be updated too or this check will start failing. `git grep` shows a few other usages in that directory, too.

Reply:
Fixed.